content/docs/user-guide/packaging/project-export/project-export-ios.md

```diff
@@ -0,0 +1,101 @@
+
+These launch parameters specified in the schema likely will be removed the next time you configure with CMake
+{{< /note >}}
+
+Now make sure to connect your iOS device to the computer. Xcode should recognize your device, and it should be compatible with the project. If not, follow the XCode error prompts to trouble shoot the issue. Updating may be required. Once this is all correct, press the play button to build the project. This can take some time.
+
+Once everything is done, assuming you used the sample configuration, you should get the following on your iOS device:
+
+{{< image-width "/images/user-guide/packaging/project-export-ios/o3de-atom-sampleviewer-ios-device-result.png" "700">}}
+
+## iOS Export Script
+O3DE ships with an experimental [iOS Export Script](https://github.com/o3de/o3de/blob/9b90a24479e2b191d2125d34c1984b013b2cb13f/scripts/o3de/ExportScripts/export_source_ios_xcode.py), capable of generating an Xcode project file to handle standard use cases of O3DE projects on iOS. This script is only designed to run from an O3DE source installation on a macOS machine.
+
+The export script has 2 primary sections: the function [`export_ios_xcode_project`](https://github.com/o3de/o3de/blob/9b90a24479e2b191d2125d34c1984b013b2cb13f/scripts/o3de/ExportScripts/export_source_ios_xcode.py#L16) and the [startup code that only runs if the script is invoked by the CLI](https://github.com/o3de/o3de/blob/9b90a24479e2b191d2125d34c1984b013b2cb13f/scripts/o3de/ExportScripts/export_source_ios_xcode.py#L73). In depth discussion on these two sections will be made in the [Developer Guide](https://docs.o3de.org/docs/engine-dev/) in the future.
+
+
+### Usage
+To use the export script, you can issue the arguments for this script at the same time that you are running the `export-project` command, so long as you are using this script as your designated export script. The arguments specific to the script will be deferred until the script begins running.
+
+The arguments are as follows:
+| Argument Name | Description | Required? |
+| - | - | - |
+| [`--script-help`](https://github.com/o3de/o3de/blob/9b90a24479e2b191d2125d34c1984b013b2cb13f/scripts/o3de/ExportScripts/export_source_ios_xcode.py#L89) | Display the help information specifically for the export script. | no |
+| [`--build-tools`](https://github.com/o3de/o3de/blob/9b90a24479e2b191d2125d34c1984b013b2cb13f/scripts/o3de/ExportScripts/export_source_ios_xcode.py#L90) | Specifies whether to build O3DE toolchain executables. This will build AssetBundlerBatch, AssetProcessorBatch. If asset tools have not been built, then assets cannot be processed. | no |
```

Comment:
I think you meant an "and" instead of a comma.
```suggestion
| [`--build-tools`](https://github.com/o3de/o3de/blob/9b90a24479e2b191d2125d34c1984b013b2cb13f/scripts/o3de/ExportScripts/export_source_ios_xcode.py#L90) | Specifies whether to build O3DE toolchain executables. This will build AssetBundlerBatch and AssetProcessorBatch. If asset tools have not been built, then assets cannot be processed. | no |
```